README.md

```diff
@@ -1,2 +1,267 @@
+  - _IPFS supports decentralized web apps built on p2p connections with power and capabilities at the edges._
+- **🤝 Supporting community growth**: We will ensure that we are setting up our community for success by addressing key needs to unlock their growth in 2019. Making our users successful and delivering on their top needs to ensure they have a smooth and productive experience with IPFS helps us retain and gain adoption, grow our community, increase our impact, and demonstrate our maturity as a project.
+  - _IPFS users are set up with the tools and capabilities they need to grow & flourish._
+- **🧠 Building the foundation for future goals**: In addition to our focused 2019 project goals, we have many [future goals](#2020-goals) for IPFS that we want to build toward. Some of the work required for those later goals requires advance planning to set up time-sensitive projects or partnerships for the future. This priority encompasses the forward-looking work we strategically take on now to set ourselves up for success later.
+  - _IPFS takes on strategic projects to stay on-track to meet future goals._
+
+**Why these priorities?**
+
+The 2019 priorities are all geared to grow our maturity as a project, increase our adoption and community, and set ourselves up for the future. To be successful, we need to level-up our ability to land complete, polished, and accessible improvements that meet our user's needs for stability, scalability, privacy, and security. We need to keep growing intelligently, both by the types of communities that can help us accelerate growth and by the number of users they can help us reach. And we need to set ourselves up with the understanding and capabilities to reach our future goals for the project and scale our impact.
+
+We designed a sorting function to help us identify which goals 1) we could focus on landing end-to-end in 2019, 2) that would bring valuable new partners and users into our community, and 3) that would set us up for future goals while delivering near-term value.
+
+**Sorting Function Explanation**
+
+> **D** = Difficulty (or "Delta" or "Distance"), **E** = Ecosystem Growth, **I** = Importance
+
+Each goal was given a score from 1 (low) - 5 (high) on each axis. We sorted first in terms of low difficulty or "delta" (i.e.  minimal additional requirements and fewer dependencies from the capabilities IPFS has now), then high ecosystem growth (growing our community and resources to help us gravity assist and accelerate our progress), and finally high importance (we want IPFS to have a strong, positive impact on the world). See the [2019 Goals (expanded)](#2019-goals-expanded) and [2020+ Goals](#2020-goals) sections for sorted priorities.
+
+## 2019 Working Groups Roadmaps
```

Comment:
This list would benefit from an introduction. Right now the doc goes from high-level goals to a list of working group links, without explaining the link.

Reply:
Good point! @olizilla would you like to volunteer and submit a PR to add that introduction?